apis/v1/storageoscluster_types.go

```diff
@@ -144,6 +144,10 @@ type StorageOSClusterSpec struct {
 
 	// Disable StorageOS scheduler extender.
 	DisableScheduler bool `json:"disableScheduler,omitempty"`
+
+	// NodeExtraEnvironment contains extra environment variables to pass
```

Comment:
nit: "NodeExtraEnvironment contains environment variables that are passed to StorageOS."

I'd be tempted to rename it to `ExtraEnvironment` or just `Environment`.  I can see us wanting to pass the same set to Node Manager or API Manager.

Reply:
Makes sense, i renamed it to `Environment`.